source/agora/consensus/ValidatorSet.d

```diff
@@ -397,6 +397,61 @@ public class ValidatorSet
         return PreImageInfo.init;
     }
 
+    /***************************************************************************
+
+        Get validator's pre-image for the given block height from the
+        validator set
+
+        Params:
+            enroll_key = The key for the enrollment in which the pre-image is
+                contained.
+            height = the desired preimage block height. If it's older than
+                     the preimage's current block height, the preimage will
+                     be hashed until the older preimage is retrieved.
```

Comment:
```suggestion
                     be hashed until the older preimage is retrieved.
                     Otherwise PreImageInfo.init is returned.
```